experimenter/tests/nimbus_integration_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+set +x
+
+export PATH=$PATH:/home/seluser/.local/bin
+
+if [[ -n "${UPDATE_FIREFOX_VERSION}" ]]; then
+    sudo ./experimenter/tests/integration/nimbus/utils/nightly-install.sh
+fi
+
+sudo apt-get -qqy update && sudo apt-get -qqy install python3-venv python3-pip
+pip install poetry --break-system-packages
+sudo chmod -R a+rwx /code/experimenter/tests/integration/
+mkdir -m a+rwx /code/experimenter/tests/integration/test-reports
+
+poetry -C experimenter/tests/integration -vvv install --no-root
+poetry -C experimenter/tests/integration -vvv run pytest --html=experimenter/tests/integration/test-reports/report.htm --self-contained-html --reruns-delay 30 --driver Firefox --verify-base-url --base-url https://nginx/nimbus/ experimenter/tests/integration/nimbus -vvv $PYTEST_ARGS
```

Comment:
-vvv is in here twice?

Reply:
One if for poetry the other for the pytest command